statistics/testdata/stats_suite_out.json

```diff
@@ -7,6 +7,417 @@
+        "├─IndexRangeScan_8(Build) 1.00 cop[tikv] table:tdouble, index:multi(b, c) range:[8 8,8 8], keep order:false",
+        "└─TableRowIDScan_9(Probe) 1.00 cop[tikv] table:tdouble keep order:false"
+      ],
+      [
+        "IndexLookUp_10 1.00 root  ",
+        "├─IndexRangeScan_8(Build) 1.00 cop[tikv] table:tdecimal, index:multi(b, c) range:[1.00000000000000000000 1.00000000000000000000,1.00000000000000000000 1.00000000000000000000], keep order:false",
+        "└─TableRowIDScan_9(Probe) 1.00 cop[tikv] table:tdecimal keep order:false"
+      ],
+      [
+        "IndexLookUp_10 1.00 root  ",
+        "├─IndexRangeScan_8(Build) 1.00 cop[tikv] table:tdecimal, index:multi(b, c) range:[4.00000000000000000000 4.00000000000000000000,4.00000000000000000000 4.00000000000000000000], keep order:false",
+        "└─TableRowIDScan_9(Probe) 1.00 cop[tikv] table:tdecimal keep order:false"
+      ],
+      [
+        "IndexLookUp_10 1.00 root  ",
+        "├─IndexRangeScan_8(Build) 1.00 cop[tikv] table:tdecimal, index:multi(b, c) range:[8.00000000000000000000 8.00000000000000000000,8.00000000000000000000 8.00000000000000000000], keep order:false",
+        "└─TableRowIDScan_9(Probe) 1.00 cop[tikv] table:tdecimal keep order:false"
+      ],
+      [
+        "TableReader_7 1.00 root  data:Selection_6",
+        "└─Selection_6 1.00 cop[tikv]  eq(test.tstring.b, \"1\"), eq(test.tstring.c, \"1\")",
+        "  └─TableFullScan_5 8.00 cop[tikv] table:tstring keep order:false"
+      ],
+      [
+        "TableReader_7 1.00 root  data:Selection_6",
+        "└─Selection_6 1.00 cop[tikv]  eq(test.tstring.b, \"4\"), eq(test.tstring.c, \"4\")",
+        "  └─TableFullScan_5 8.00 cop[tikv] table:tstring keep order:false"
+      ],
+      [
+        "TableReader_7 1.00 root  data:Selection_6",
+        "└─Selection_6 1.00 cop[tikv]  eq(test.tstring.b, \"8\"), eq(test.tstring.c, \"8\")",
+        "  └─TableFullScan_5 8.00 cop[tikv] table:tstring keep order:false"
+      ],
+      [
+        "TableReader_7 1.00 root  data:Selection_6",
+        "└─Selection_6 1.00 cop[tikv]  eq(test.tdatetime.b, 2001-01-01 00:00:00.000000), eq(test.tdatetime.c, 2001-01-01 00:00:00.000000)",
+        "  └─TableFullScan_5 4.00 cop[tikv] table:tdatetime keep order:false"
+      ],
+      [
+        "TableReader_7 1.00 root  data:Selection_6",
+        "└─Selection_6 1.00 cop[tikv]  eq(test.tdatetime.b, 2001-01-02 00:00:00.000000), eq(test.tdatetime.c, 2001-01-02 00:00:00.000000)",
+        "  └─TableFullScan_5 4.00 cop[tikv] table:tdatetime keep order:false"
+      ],
+      [
+        "TableReader_7 1.00 root  data:Selection_6",
+        "└─Selection_6 1.00 cop[tikv]  eq(test.tdatetime.b, 2001-01-04 00:00:00.000000), eq(test.tdatetime.c, 2001-01-04 00:00:00.000000)",
+        "  └─TableFullScan_5 4.00 cop[tikv] table:tdatetime keep order:false"
+      ],
+      [
+        "TableReader_7 1.33 root  data:Selection_6",
+        "└─Selection_6 1.33 cop[tikv]  eq(test.tint.b, 1)",
```

Comment:
A little strange...This should be `1.00`.

Reply:
It is calculated in `Index.BetweenRowCount`. 1.33 = 1.00(from TopN) + 0.33(from Hist), it is not accurate but it's better to optimize it later in another PR.